### PR TITLE
Update CONTRIBUTING.md to introduce Issue Tracker

### PR DESCRIPTION
**Pull request by mizuni**
Also made some template changes for issue filing

---
This change is [Reviewable](https://reviewable.io/reviews/googlei18n/libphonenumber/1670)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,15 +29,19 @@ Please copy this template into your report and answer the questions to the best
...
 ``` nomarkdown
-*   Country/region affected (e.g., "US"):
-*   Example number(s) affected ("+1 555 555-1234"):
-*   The phone number range(s) to which the issue applies ("+1 555 555-XXXX"):
-*   The type of the number(s) ("fixed-line", "mobile", "short code", etc.):
-*   The cost, if applicable ("toll-free", "premium rate", "shared cost"):
-*   Supporting evidence (for example, national numbering plan, announcement from mobile carrier, news article):
+*   Country:
+*   Example number(s) and/or range(s):
+*   For short codes, cost and dialing restrictions:
```
Comment (lararennie) [inline CONTRIBUTING.md]: Actually, I guess we still need to ask the type of the number.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,15 +29,19 @@ Please copy this template into your report and answer the questions to the best
...
+*   Where or whom did you get the number(s) from:
+*   Authoritative evidence (e.g. national numbering plan, operator annoucement):
```
Comment (lararennie) [inline CONTRIBUTING.md]: annoucement -> announcement

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -29,15 +29,19 @@ Please copy this template into your report and answer the questions to the best
...
+*   Link from demo (http://libphonenumber.appspot.com) showing error:
 ```
 
-### Provide supporting evidence
+For issues affecting validation, formatting, or other aspects covered by the metadata,
+we invite you to file issues in Google new [Issue Tracker](issuetracker.google.com/issues/new?component=192347),
```
Comment (lararennie) [inline CONTRIBUTING.md]: Google -> Google's

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -29,15 +29,20 @@ Please copy this template into your report and answer the questions to the best
...
+*   Number type ("fixed-line", "mobile", "short code", etc.):
+*   For short codes, cost and dialing restrictions:
+*   Where or whom did you get the number(s) from:
+*   Authoritative evidence (e.g. national numbering plan, operator announcement):
+*   Link from demo (http://libphonenumber.appspot.com) showing error:
 ```
 
-### Provide supporting evidence
+For issues affecting validation, formatting, or other aspects covered by the metadata,
+we invite you to file issues in Google's new [Issue Tracker](issuetracker.google.com/issues/new?component=192347),
```
Comment (giggsey) [inline CONTRIBUTING.md]: The link is missing the protocol, so GitHub is rendering it as relative.

**Pull request merged**